4_processing_features/extract_sc_features_dp.py

```diff
@@ -0,0 +1,209 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Process single cell morphology features for DeepProfiler readouts
+
+# ## Import libraries
+
+# In[1]:
+
+
+import pathlib
+import pandas as pd
```

Comment:
Consider removing this pandas import if it is not needed.

Reply:
👍 Great catch ty!